features/delivery.feature

```diff
@@ -107,6 +107,8 @@ Feature: Delivery of errors
     Given I set the HTTP status code for the next request to 500
     And I run "AutoSessionScenario"
     And I wait to receive a session
+    And I background the app for 0 seconds
+    And I wait for 10 seconds
```

Comment:
I think these are debug lines no longer needed?  The tests are failing because the first isn't defined.

Reply:
Yup, no longer needed.